src/main/java/org/venice/beachfront/bfapi/BfApiConfig.java

```diff
@@ -241,12 +265,16 @@ private boolean isPublicEndpoint(String path) {
 		@Autowired
 		private ApiKeyAuthProvider apiKeyAuthProvider;
 		@Autowired
+		private JWTAuthProvider jwtAuthProvider;
+		@Autowired
 		private FailedAuthEntryPoint failureEntryPoint;
+		@Value("${jwt.enabled}")
```

Comment:
Consider adding a default value to the code if the property is missing. E.g.:

`@Value("${jwt.enabled:false}")`

Reply:
as above: application.properties default value